api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreConstants.java

```diff
@@ -61,4 +61,6 @@
 	public static final String GP_MPI_ACCESS_PASSWORD = "registrationcore.mpi.password";
 
 	public static final String GP_PROBABLY_MATCH_ENABLED = "registrationcore.openempi.enableProbabilisticMatching";
+	
+	public static final String GP_BIRTHDATE_ESTIMATION_MONTH = "registrationcore.birthdateEstimationMonth";
```

Comment:
I would also rename GP_BIRTHDATE_ESTIMATION_MONTH to GP_BIRTHDATE_ESTIMATION_START_MONTH